rosflight_sim/src/sil_board.cpp

```diff
@@ -364,45 +372,54 @@ float SIL_Board::sonar_read(void)
     return alt + sonar_stdev_*normal_distribution_(random_generator_);
 }
 
-// PWM
-void SIL_Board::pwm_init(bool cppm, uint32_t refresh_rate, uint16_t idle_pwm)
+// RC
+void SIL_Board::rc_init(rc_type_t rc_type)
 {
+  (void)rc_type;
   rc_received_ = false;
   latestRC_.values[0] = 1500; // x
   latestRC_.values[1] = 1500; // y
   latestRC_.values[3] = 1500; // z
   latestRC_.values[2] = 1000; // F
-  latestRC_.values[4] = 1000; // attitude override
+  latestRC_.values[4] = 1000; // override
   latestRC_.values[5] = 1000; // arm
-
-  for (size_t i = 0; i < 14; i++)
-    pwm_outputs_[i] = 1000;
-
-  rc_sub_ = nh_->subscribe("RC", 1, &SIL_Board::RCCallback, this);
```

Comment:
This line shouldn't have moved, right? We're subscribing to an RC input so it's still specific to `rc_init()`, not the new `pwm_init()` function